### DISCUSSION TITLE
Save to WAV

### DISCUSSION THREAD

**Discussion by thweasel** [oxidaan/zqloader#3] [Ideas]
Hey, found your program it's awesome. 

It would be nice if the zqloader could output to a WAV file instead of the sound card so it can be played back from other digital playback devices.

Cheers,
Owen

**Comment by oxidaan** [comment]
Just released a version that can generate wav (and tzx) files!

**Reply by thweasel** [reply]
Cheers, I will have a look and let you know how I get on :)
